README.md

```diff
@@ -1,53 +1,50 @@
 <p align="center">
-<a href="https://pi-hole.net"><img src="https://camo.githubusercontent.com/578cfb146b0bb867e3609c5581761cf0b845b527/68747470733a2f2f66697265626f672e6e65742f566f72746578322e706e67" width="150" height="255"></a><br/>
-<b>Network-wide ad blocking via your own Linux hardware</b>
+<a href="https://pi-hole.net"><img src="https://pi-hole.github.io/graphics/Vortex/Vortex_with_text.png" width="150" height="255" alt="Pi-hole"></a><br/>
+<b>Network-wide ad blocking via your own Linux hardware</b><br/>
 </p>
 
-The Pi-hole blocks ads via a [DNS sinkhole](https://en.wikipedia.org/wiki/DNS_Sinkhole), so all your devices are protected without the need to install client-side software.
+The Pi-hole is a [DNS sinkhole](https://en.wikipedia.org/wiki/DNS_Sinkhole) that protects your devices from unwanted content, without installing any client-side software.
 
 - **Easy-to-install**: our versatile installer walks you through the process, and [takes less than ten minutes](https://www.youtube.com/watch?v=vKWjx1AQYgs)
-- **Resolute**: ads are blocked in _non-browser locations_ such as ad-laden mobile apps and smart TVs
-- **Fast**: speeds up the feel of everyday browsing by caching DNS queries, saving bandwidth
-- **Lightweight**: runs smoothly and requires [minimal resources](https://discourse.pi-hole.net/t/hardware-software-requirements/273)
-- **Robust**: a command line interface for those preferring CLI, and/or wanting to automate tasks
-- **Informative**: a beautiful and secure Web Interface dashboard to control your Pi-hole
-- **Versatile**: can optionally function as a DHCP server, ensuring your devices will not need configuring individually
-- **Scalable**: [capable of handling hundreds of millions of queries](https://pi-hole.net/2017/05/24/how-much-traffic-can-pi-hole-handle/) when installed on powerful hardware
+- **Resolute**: content is blocked in _non-browser locations_, such as ad-laden mobile apps and smart TVs
```

Comment:
Should the sentences start with capital letters, maybe?  e.g:
`**Resolute**: Content is blocked...`

Reply:
I think the extra capitalisation would make it look clumsy, considering the bold is the start of the sentence 😄